lib/k10/dtb.ex

```diff
@@ -297,16 +295,8 @@ defmodule K10.DTB do
     {:fdt_end, <<>>}
   end
 
-  defp eat_nulls(rest, 0) do
-    rest
-  end
-
-  defp eat_nulls(<<0::8, rest::binary>>, cnt) do
-    eat_nulls(rest, cnt - 1)
-  end
-
-  # smallest v >= n such that v is a multiple of k
-  defp align(n, k) do
-    div(n + k - 1, k) * k
+  defp align(data, offset, alignment) do
```

Comment:
np: could you add a comment that explains what this function is meant to do?

Reply:
Oh yeah fair point! I forgot to rewrite the comment :face_exhaling: